### ISSUE TITLE
Ensure the scope of this repository is accurate and clear to the public

### ISSUE THREAD

**Issue by RubenVerborgh** [solid/specification#133]
As it is turning out, this repository contains multiple orthogonal specifications.

However, I see people on several places being confused regarding the nature of this repository, not in the least ourselves:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/675313/70861449-7b8cc080-1f2e-11ea-9dc7-1eaee26a7dc2.png">

There are multiple documents that could be understood as "_the_ specification", so I'm afraid this will lead to confusion.

Shall we use this issue to agree on what we want that scope to be, and reflect this in:
- a description in the README
- a renaming of this repository to reflect this (perhaps `solid/specifications`)?
  - put redirects in place so nothing breaks

**Comment by csarven** [comment]
tl;dr: I don't object to the use of the term "specifications", but the change from "specification". If people are confused about this repo's purpose or scope, or what's "the specification", we should first start with clarifying the description in the README, the ecosystem doc etc.

Part of the confusion is due to constant renaming/redirecting/reshuffling.. as new people join the Solid project and evidently wants to shape it to their understanding - which is welcome.. there is always room to evolve and converge - and so we've been adding to the pile of repos and references. FWIW, there was nothing wrong with "solid-spec", and "specification" didn't clarify anything above and beyond, but we went along with it any way due to other reasons to departure from the earlier work. Here we are again, this time questioning whether the plural would be more appropriate for a repo URI to capture the essence of everything that's under the ecosystem. I doubt that the "s" is going to single-handedly will fix that. In any case, why is this a surprise now? Certainly wasn't when the repo was created. Even after https://lists.w3.org/Archives/Public/public-solid/2019May/0015.html proposed adopting the notion of ecosystem and orthogonal specs - which has been generally agreed and adopted by this repo/project.

**Comment by RubenVerborgh** [comment]
Thanks @csarven, I completelly agree and will scope this issue more broadly.

**Comment by csarven** [comment]
We can perhaps borrow some ideas from:
* https://www.w3.org/wiki/Open_Web_Platform
* https://w3c.github.io/web-annotation/
* https://www.w3.org/TR/sparql11-overview/
..

**Comment by RubenVerborgh** [comment]
Let's tackle this in #141.
